pages/linux/addpart.md

```diff
@@ -0,0 +1,7 @@
+# addpart
+
+> Tells the Linux kernel about the existence of the specified partition. The command is a simple wrapper around the "add partition" ioctl, [Docs](https://linux.die.net/man/8/addpart).
```

Comment:
```suggestion
> Tells the Linux kernel about the existence of the specified partition.
> The command is a simple wrapper around the `add partition` ioctl.
> More information: <https://linux.die.net/man/8/addpart>.
```

Reply:
Thank you for the suggested changes, it looks much cleaner now.